lib/miq_expression/field.rb

```diff
@@ -77,6 +77,10 @@ def arel_attribute
     target.arel_attribute(column)
   end
 
+  def report_column
```

Comment:
Although I like this name *way* better, I'm just thinking that these are used in other parts of the UI than reporting - anywhere there is an expression builder. I wonder if there could be a more generic name for this. @gtanzillo do you have any ideas?